.github/workflows/release.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: 1.21
+      - name: Format code
```

Comment:
Do we need to run the format step in the release workflow?

Reply:
I suppose note, but it does assume no one has made any mistakes on `main`